python_modules/dagster/dagster/core/instance/config.py

```diff
@@ -92,4 +92,5 @@ def dagster_instance_config_schema():
         "run_launcher": config_field_for_configurable_class(),
         "telemetry": Field({"enabled": Field(Bool, is_required=False)}),
         "instance_class": config_field_for_configurable_class(),
+        "python_logs": Field(dict, is_required=False),
```

Comment:
just referencing the previous comment, this can be something more specific than "dict"